src/pages/før-du-begynner/FørDuBegynner.tsx

```diff
@@ -62,21 +62,25 @@ const FørDuBegynner: FunctionComponent<RouteComponentProps> = props => {
     };
 
     const sjekkEksistensOgRedirect = async () => {
-        try {
+
             // TODO: Fjern når vi fjerner default fnr
             if (fnr === '00000000000') {
                 throw new Error();
```

Comment:
Appen krasjer om man skriver inn 00000000000 som fnr pga try-catchen er fjernet.
Kanskje vi bare burde fjerne default fnr her... Eller bare ha default fnr om vi kjører i test/mock.
@kjesvale hva var tanken her?

Reply:
Vi trenger vel fremdeles try-catchen. Når vi kjører `hentKandidat(fnr)` vil den returnere en exception hvis kandidaten ikkefinnes fra før, og da skal bruker redirectes til Registrering i stedet for endring. Det er lagt til et unntak for "000..." fordi man da alltid skal gå til endresiden. Men burde nok sjekke om vi mocker her, ja.

```
// FørDuBegynner.txt
const appIsMocked = process.env.REACT_APP_MOCK;
...
const [fnr, setFnr] = useState<string>(appIsMocked ? '00000000000' : '');
...
if (appIsMocked && fnr === '00000000000') {
    throw new Error();
}
```